addons/iap/__manifest__.py

```diff
@@ -11,13 +11,15 @@
 to support In-App purchases inside Odoo. """,
     'depends': [
         'web',
-        'base_setup'
+        'base_setup',
+        'mail',
```

Comment:
IAP was intended to work without mail. If you do that, you can also merge iap_mail into iap.

Reply:
Hello @tde-banana-odoo ,
> IAP was intended to work without mail.

I didn't know that, why is that ? :eyes:
The only reason the dependency is added is to add the chatter to the `iap.account` model, so we could easily move that to `iap_mail` and avoid the dependency change.

Reply:
Don't know (not iap author, just had to work with iap and iap_mail sometimes), I suppose IAP was coded to be as small as possible without any dependencies to let people do stuff with it. Don't have much opinion on it, just pointing out that iap_mail exists. If there is no real reason to keep iap and iap_mail we could think of merging both.